### PR TITLE
Update README.md and RELEASE.md for 0.19.1

### PR DESCRIPTION
**Pull request by yongtang**
This PR is in preparation for release 0.19.1

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

**Comment by yongtang** [comment]
FYI @casassg

File: RELEASE.md [hunk 1]
```diff
@@ -1,3 +1,17 @@
+# Release 0.19.1
+
+## Bug Fixes
+* Fixed wheel issue with tensorflow-io-gcs-filesystem by removing build temporary folder.
+
+## Thanks to our Contributors
+
+This release contains contributions from many people:
+
+Gerard Casas Sae, Kota Yamaguchi, Vignesh Kothapalli, Vo Van Nghia, Yong Tang, emkornfield
```
Comment (casassg) [inline RELEASE.md:10]:
```suggestion
Gerard Casas Saez, Kota Yamaguchi, Vignesh Kothapalli, Vo Van Nghia, Yong Tang, emkornfield
```

**Review by casassg — APPROVED** [review]

File: RELEASE.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
+# Release 0.19.1
+
+## Bug Fixes
+* Fixed wheel issue with tensorflow-io-gcs-filesystem by removing build temporary folder.
```
Comment (kvignesh1420) [inline RELEASE.md:4]: should be "removing temporary build folder"

Comment (yongtang) [reply RELEASE.md]: Thanks @kvignesh1420 , the PR has been updated.

**Review by kvignesh1420 — APPROVED** [review]
LGTM.

**Comment by yongtang** [comment]
After merged to R0.19 branch, the Windows build is failing due to incompatible abseil cpp library. I am not exactly sure why this is the case (not happening in Linux or macOS). I will try to add an empty commit to R0.19 to give an additional try.

**Comment by casassg** [comment]
Seems it continues to fail: https://github.com/tensorflow/io/runs/3146855788

**Comment by yongtang** [comment]
The issue seems to be coming from the bazel cache having trouble on Windows.

This has been resolved now with bazel cache disabled on Windows:

https://github.com/tensorflow/io/actions/runs/1062520710

Will cut the release tomorrow.

**Comment by yongtang** [comment]
Released binary:
```
3ce0c715bf574681789b359e72bd256503ae04b8695cefdc318164e6e8027a4e  tensorflow_io-0.19.1-cp36-cp36m-macosx_10_14_x86_64.whl
5edfb6cfb593449a432d4c32c1a889f79afee78b2745514be55c14bfd767827c  tensorflow_io-0.19.1-cp36-cp36m-manylinux_2_12_x86_64.manylinux2010_x86_64.whl
c8e081b838aa779d085b2c4ba24966ae04796a2650349025f569bda86ceab04f  tensorflow_io-0.19.1-cp36-cp36m-win_amd64.whl
2cb37d144babfc39841561b6f001bd876afef425a39ebff4ab4de28240c43b5d  tensorflow_io-0.19.1-cp37-cp37m-macosx_10_14_x86_64.whl
373f4d75d6c994163bb0ac63e5f920fb7b9be8037aa3c9dd4e6535de172f177a  tensorflow_io-0.19.1-cp37-cp37m-manylinux_2_12_x86_64.manylinux2010_x86_64.whl
07a74db49c3bc42a8421549e7b4a3121ddbc41300bb9a929bfc6105be500c6fc  tensorflow_io-0.19.1-cp37-cp37m-win_amd64.whl
31f52ed6ced3ee9440d2d93c811b681da4ce33a75132c66e6e512503dfd68f16  tensorflow_io-0.19.1-cp38-cp38-macosx_10_14_x86_64.whl
e74af766dcbe56aca1ac74bfa45ed6d10a4650832f115bee52f112af336195ff  tensorflow_io-0.19.1-cp38-cp38-manylinux_2_12_x86_64.manylinux2010_x86_64.whl
aa9a7cc5a21b20a7d1e79d1597181128b029d3ff9b091f66691b87bc242fdcdd  tensorflow_io-0.19.1-cp38-cp38-win_amd64.whl
c081a7b928988dc0658c622b306bf312cdff8fdfdccfb1102cd48dc8f373c49f  tensorflow_io-0.19.1-cp39-cp39-macosx_10_14_x86_64.whl
49905b251cbb8dade36678f2b45b4a57a667b6d81e10ec3d66aed71ed70a826c  tensorflow_io-0.19.1-cp39-cp39-manylinux_2_12_x86_64.manylinux2010_x86_64.whl
742418a4d9f6e4e65a3f2f60d564c4ec22859573a43c7fb47db848b60752aff5  tensorflow_io-0.19.1-cp39-cp39-win_amd64.whl
01d79022cd385f92adc7861f3d99d86c328ef116c2795e125754cac11780c607  tensorflow_io_gcs_filesystem-0.19.1-cp36-cp36m-macosx_10_14_x86_64.whl
3d7dbf377d185793d5643256ef80d869b21af2be76af1117bb1bc2ab8c5d5258  tensorflow_io_gcs_filesystem-0.19.1-cp36-cp36m-manylinux_2_12_x86_64.manylinux2010_x86_64.whl
a61924fd6ae845974dda1f5962a9693ff14a84879a6730d0c858ea3a2045944e  tensorflow_io_gcs_filesystem-0.19.1-cp36-cp36m-win_amd64.whl
ac155105ee2b1a28016aaf04a9783ab15aaebc407f7fe9886bf0405f5fa233d9  tensorflow_io_gcs_filesystem-0.19.1-cp37-cp37m-macosx_10_14_x86_64.whl
9cd718af3f51d08886ca862ee0d3f1696ebea146cf9826b93edc946c053e0ddb  tensorflow_io_gcs_filesystem-0.19.1-cp37-cp37m-manylinux_2_12_x86_64.manylinux2010_x86_64.whl
77186ef42fe85119ccd97356065acd1d69651200160d944bda0711b48e9cce96  tensorflow_io_gcs_filesystem-0.19.1-cp37-cp37m-win_amd64.whl
70b56e1a633493453e15d614c42c4cc7e7c4b7092496666b763a6fc3cf96a4c4  tensorflow_io_gcs_filesystem-0.19.1-cp38-cp38-macosx_10_14_x86_64.whl
c852088556a07ae0a81639bda367861efd4e46fa21740faad3d16a3c95f6f11c  tensorflow_io_gcs_filesystem-0.19.1-cp38-cp38-manylinux_2_12_x86_64.manylinux2010_x86_64.whl
b2d5fa7b8d65464cbd2b52274135a474a4d81a91c895349c7cbf1dda6d45d8d8  tensorflow_io_gcs_filesystem-0.19.1-cp38-cp38-win_amd64.whl
190a432548de5ad2ef1d5ffaafafc3297ffa8e355baa7a54d1ff9dd4bdad4223  tensorflow_io_gcs_filesystem-0.19.1-cp39-cp39-macosx_10_14_x86_64.whl
1ca149fa2874e768558540b3e9059b9d50cc86f2683933c165a4ca54769080a4  tensorflow_io_gcs_filesystem-0.19.1-cp39-cp39-manylinux_2_12_x86_64.manylinux2010_x86_64.whl
47c0e35f4c591b7894ca2361184528f7b30f3cbf134634f117b0577e47aa8a43  tensorflow_io_gcs_filesystem-0.19.1-cp39-cp39-win_amd64.whl
```

**Comment by yongtang** [comment]
Releases of 0.19.1 is available in:

https://pypi.org/project/tensorflow-io/0.19.1/
https://pypi.org/project/tensorflow-io-gcs-filesystem/0.19.1/

https://github.com/tensorflow/io/releases/tag/v0.19.1

**Comment by kvignesh1420** [comment]
Thanks @yongtang

**Comment by kvignesh1420** [comment]
@casassg can you please verify and let us know if your issue is fixed?

**Comment by casassg** [comment]
@kvignesh1420 @yongtang Seems its fixed now.

```
pip install tensorflow-io-gcs-filesystem==0.19.1
Collecting tensorflow-io-gcs-filesystem==0.19.1
  Downloading tensorflow_io_gcs_filesystem-0.19.1-cp37-cp37m-macosx_10_14_x86_64.whl (1.6 MB)
     |████████████████████████████████| 1.6 MB 1.0 MB/s
Installing collected packages: tensorflow-io-gcs-filesystem
Successfully installed tensorflow-io-gcs-filesystem-0.19.1
WARNING: You are using pip version 20.1.1; however, version 21.2.1 is available.
You should consider upgrading via the '/Users/gcasassaez/workspace/local_envs/tfio_fix/env/bin/python3 -m pip install --upgrade pip' command.
(env) tfio_fix|⇒ cat env/lib/python3.7/site-packages/
```
The folder now only has tfio_gcs_filesystem

**Pull request merged**